app/views/passengers/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="container">
+  <div class="div3">
```

Comment:
In general, class names like `div3` aren't so great-- I have no idea what they represent or what they'll hold! When I add styles to this in the CSS file, how do I know what this is supposed to look like by the name?